src/System.Memory/ref/System.Memory.cs

```diff
@@ -91,13 +72,20 @@ public partial struct Span<T>
         public bool TryCopyTo(System.Span<T> destination) { throw null; }
     }
 
-    public static class SpanExtensions
+    public static class Span
```

Comment:
Actually I believe the assembly version comes from System.###\dir.props, not from the csproj files, as done in other places. Probably should add that file and remove the AssemblyVersion property from the two csproj files.